### PR TITLE
Switched PHP requirements to 5.4

### PR DESCRIPTION
**Pull request by COil**
This PR allows to use the bot-detect with a PHP 5.4 installation. It forces the composer platform so the Symfony components versions don't have to be changed. When dropping the PHP 5.4 support, just delete the config platform key.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,9 +13,9 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "php": ">=5.5.9",
-        "symfony/yaml": "^2.3|^3.0",
-        "symfony/config": "^2.3|^3.0"
```
Comment (lennerd) [inline composer.json]: Why did you change the versions for the Symfony dependencies?

Comment (COil) [inline composer.json]: Weird, the OK are ok on my computer but not in travis, if I revert the Symfony deps, the tests don't pass anymore, can you check ?

Comment (COil) [inline composer.json]: It's Ok on my computer, Travis doesn't behave the same way.

Comment (COil) [inline composer.json]: Seems OK now ?

**Comment by COil** [comment]
@lennerd  Ping  :)

**Comment by lennerd** [comment]
Sorry.

Can you please add some information as a comment to this PR explaining your changes to the composer file?

**Comment by COil** [comment]
@lennerd done

**Comment by lennerd** [comment]
Sorry, I mean as comment here, on this page. I'm not sure, why you added the config part and what it is for.

**Comment by COil** [comment]
The config part is to lock the requirements on PHP 5.4, even the individual requirement of each composer package allows to use another major version. (php 5.5)

**Comment by COil** [comment]
Any chance to have a tag today ?

**Comment by lennerd** [comment]
Sorry, I thought I responded to this. Must be a late night no concentration left issue. And I don't want to make something wrong when reverting this.

I'm not quite sure, what it means to lock PHP to 5.4. Wouldn't it make more sense to test the Symfony dependencies with the appropriate PHP version? Maybe that's possible with Travis somehow.

**Comment by COil** [comment]
But you told me not to change to Symfony components requirements... 

"Why did you change the versions for the Symfony dependencies?"

**Comment by lennerd** [comment]
Sorry, I misunderstood the meaining of the composer file changes.

Thanks for also locking the version of the coveralls library. Hope this works for your version of PHP now.
This again is a great example why we should split up the bot archive and the library itself.

**Comment by COil** [comment]
Thank you. Seems OK. :+1:
